packages/bolt-matrix/mod.ts

```diff
@@ -59,16 +42,14 @@ export default class MatrixPlugin extends BoltPlugin {
 	async start(bolt: Bolt) {
 		this.bolt = bolt;
 		if (!existsSync(this.config.reg_path)) {
-			const reg = new AppServiceRegistration(this.config.homeserverUrl);
+			const reg = new AppServiceRegistration(this.config.appserviceUrl);
 			reg.setAppServiceToken(AppServiceRegistration.generateToken());
 			reg.setHomeserverToken(AppServiceRegistration.generateToken());
-			reg.setId(
-				'b4d15f02f7e406db25563c1a74ac78863dc4fbcc5595db8d835f6ee6ffef1448'
-			);
+			reg.setId(AppServiceRegistration.generateToken());
 			reg.setProtocols(['bolt']);
 			reg.setRateLimited(false);
-			reg.setSenderLocalpart('boltbot');
-			reg.addRegexPattern('users', '@bolt_*', true);
+			reg.setSenderLocalpart('bot.bolt');
+			reg.addRegexPattern('users', `@bolt-(discord|revolt)_.+:${this.config.domain}`, true);
```

Comment:
is there any reason for the change to the local part? convention i'm not aware of? aesthetics?

also, don't do the `(discord|revolt)` part of the regex, that doesn't work since guilded support also exists and other platforms could be added in the future.

```suggestion
			reg.setSenderLocalpart('bot.bolt');
			reg.addRegexPattern('users', `@bolt-.+_.+:${this.config.domain}`, true);
```

Reply:
> bot.bolt

personal preference, but i guess we can make it configurable

> regex pattern

because how you make username is `@bolt-${platform}_${id}:${homeserver}`, i simply made it match this pattern